src/content/ui/navigation/index.md

```diff
@@ -50,20 +48,17 @@ Applications with simple navigation and deep linking requirements can use the
 `Navigator` for navigation and the [`MaterialApp.routes`][] parameter for deep
 links:
 
+<?code-excerpt "ui/navigation/lib/navigator_named_routes.dart (push-route)"?>
```

Comment:
I like this shows how to use the `Navigator` API a bit more, but was it intentional to lose how to declare the routes? The preceding text mentions the `MaterialApp.routes` parameter, but now we don't show it.

Perhaps we could show both while still keeping the sample short or have two snippets right next to each other?

Reply:
This was intentional for a few reasons:

1. It's much easier for me to see the subtle differences between these route types when the focus is on the `onPressed` snippet.
2. Before this change we were already calling out parts that weren't in the snippets, so I figured it should be okay to continue this practice (see `BuildContext` references).
3. We have links to full examples that go into the smaller details that are required for each routing type.
4. Some of these routing types have quite a lot of parts and I wasn't able to get these various parts to look good in the generated snippets without mostly adding the full code snippets. I also was not able to get the formatting to look great. So in the end, I just really liked the simplified layout and would like to stay with it if possible.